src/app/components/loadDataModal.js

```diff
@@ -0,0 +1,74 @@
+import nunjucks from 'nunjucks';
+import axios from 'axios';
+import countProgress from '../base/countProgress';
+import notify from './../base/notify';
+import { notes, messages } from './../constants/constants';
+
+export default class LoadDataModal {
+    constructor(container) {
+        this.container = container;
+        this.modalCloseButton = this.container.querySelector('.js-history-modal-close');
+        this.modalContent = this.container.querySelector('.js-history-modal-content');
+        this.modalWindow = document.querySelector('.js-history-modal');
+        this.historyButtons = [...document.querySelectorAll('.js-history')]; // select all history buttons
+
+        // get name of template to render
+        const jsTemplates = 'http://localhost:5001/js/templates';
```

Comment:
this would be better in `constants.js` since it doesn't change and then it's reusable in any `js` file